### ISSUE TITLE
arm64 apt package missing for datadog-apm-library-dotnet

### ISSUE THREAD

**Issue by endocrimes** [DataDog/dd-trace-dotnet#4135]
**Describe the bug**
On Ubuntu + amd64 one can install the datadog dotnet tracing library via apt and the datadog apt repository, unfortunately it seems like the package is missing on arm64.

**To Reproduce**

1: Setup the datadog apt repository on an arm64 system:

```
sudo sh -c "echo 'deb [signed-by=/usr/share/keyrings/datadog-archive-keyring.gpg] https://apt.datadoghq.com/ stable 7' > /etc/apt/sources.list.d/datadog.list"
sudo touch /usr/share/keyrings/datadog-archive-keyring.gpg
sudo chmod a+r /usr/share/keyrings/datadog-archive-keyring.gpg

curl https://keys.datadoghq.com/DATADOG_APT_KEY_CURRENT.public | sudo gpg --no-default-keyring --keyring /usr/share/keyrings/datadog-archive-keyring.gpg --import --batch
curl https://keys.datadoghq.com/DATADOG_APT_KEY_382E94DE.public | sudo gpg --no-default-keyring --keyring /usr/share/keyrings/datadog-archive-keyring.gpg --import --batch
curl https://keys.datadoghq.com/DATADOG_APT_KEY_F14F620E.public | sudo gpg --no-default-keyring --keyring /usr/share/keyrings/datadog-archive-keyring.gpg --import --batch
```

2: Attempt to install the tracer

```
sudo apt-get update
sudo apt-get install datadog-agent datadog-signing-keys datadog-apm-library-dotnet
```

**Expected behavior**

The package should be available like it is on amd64:

```
amazon-ebs.ubuntu22-amd64-ami: Reading package lists...
amazon-ebs.ubuntu22-amd64-ami: Reading package lists...
amazon-ebs.ubuntu22-amd64-ami: Building dependency tree...
amazon-ebs.ubuntu22-amd64-ami: Reading state information...
amazon-ebs.ubuntu22-amd64-ami: The following NEW packages will be installed:
amazon-ebs.ubuntu22-amd64-ami:   datadog-agent datadog-apm-library-dotnet datadog-signing-keys
amazon-ebs.ubuntu22-amd64-ami: 0 upgraded, 3 newly installed, 0 to remove and 3 not upgraded.
amazon-ebs.ubuntu22-amd64-ami: Need to get 370 MB of archives.
amazon-ebs.ubuntu22-amd64-ami: After this operation, 1159 MB of additional disk space will be used.
amazon-ebs.ubuntu22-amd64-ami: Get:1 https://apt.datadoghq.com/ stable/7 amd64 datadog-apm-library-dotnet amd64 2.27.0-1 [13.8 MB]
amazon-ebs.ubuntu22-amd64-ami: Get:2 https://apt.datadoghq.com/ stable/7 amd64 datadog-agent amd64 1:7.44.0-1 [356 MB]
```

**Screenshots**

```
 E: Unable to locate package datadog-apm-library-dotnet
```

**Runtime environment (please complete the following information):**
- Instrumentation mode: Datadog Apt Repository
- Tracer version: Any
- OS: Ubuntu 22.04
- CLR: [e.g. .NET Framework 4.6.2, .NET Core 2.1]

**Additional context**

**Comment by lucaspimentel** [comment]
Thanks for reporting this, @endocrimes. I can confirm that an arm64 package for `datadog-apm-library-dotnet` has not being published yet. We do plan on supporting library injection on arm64 (as we do with our other Linux packages), but I don't have a timeline at this time.

**Comment by endocrimes** [comment]
No worries! (This is at least an easy way to track it 😅)

I've flagged out installing them on arm for now - it just means our dotnet things will be stuck on amd64 for a while 😅

**Comment by leongold2** [comment]
E: Unable to locate package datadog-apm-inject
E: Unable to locate package datadog-apm-library-all

seem to be missing as well. any workarounds? without these libs, i assume there's no APM available?

cc @lucaspimentel, thanks in advance

**Comment by andrewlock** [comment]
I believe this was fixed in version 2.42.0 - could you give it another try and make sure it works for you? 🙂  Thanks!
